src/player.h

```diff
@@ -511,6 +513,11 @@ class Player final : public Creature, public Cylinder
 		void addConditionSuppressions(uint32_t conditions);
 		void removeConditionSuppressions(uint32_t conditions);
 
+		Reward* getReward(uint32_t rewardId, bool autoCreate);
```

Comment:
There is a point. Calling ````getReward(rewardId, false)```` won't give the player a reward if he doesn't have one already, i.e the player tries to open the boss' corpse but didn't help in the fight, if there was no ````autoCreate```` parameter an empty reward container would be presented to the player in the boss corpse and in the reward chest.